www/calendar/inner.js

```diff
@@ -818,106 +818,130 @@ define([
         // Show calendars
         var calendars = h('div.cp-calendar-list');
         var $calendars = APP.$calendars = $(calendars).appendTo($container);
+        var isMobileView = window.innerWidth <= 600;
+        var visible = !isMobileView;  // Initialize 'visible' state: true for large screens, false for mobile
+
         onCalendarsUpdate.reg(function () {
             $calendars.empty();
             var privateData = metadataMgr.getPrivateData();
-            var filter = function (teamId) {
+
+            var filter = (teamId) => {
                 var LOOKUP = {};
-                return Object.keys(APP.calendars || {}).filter(function (id) {
+                return Object.keys(APP.calendars || {}).filter((id) => {
                     var cal = APP.calendars[id] || {};
-                    var teams = (cal.teams || []).map(function (tId) { return Number(tId); });
-                    return teams.indexOf(typeof(teamId) !== "undefined" ? Number(teamId) : 1) !== -1;
-                }).map(function (k) {
-                    // nearly constant-time pre-sort
+                    var teams = (cal.teams || []).map((tId) => Number(tId));
+                    return teams.indexOf(typeof (teamId) !== "undefined" ? Number(teamId) : 1) !== -1;
+                }).map((k) => {
                     var cal = APP.calendars[k] || {};
                     var title = Util.find(cal, ['content', 'metadata', 'title']) || '';
                     LOOKUP[k] = title;
                     return k;
-                }).sort(function (a, b) {
+                }).sort((a, b) => {
                     var t1 = LOOKUP[a];
                     var t2 = LOOKUP[b];
                     return t1 > t2 ? 1 : (t1 === t2 ? 0 : -1);
                 });
             };
-            var tempCalendars = filter(0);
-            if (tempCalendars.length && tempCalendars[0] === APP.currentCalendar) {
-                APP.$calendars.append(h('div.cp-calendar-team', [
-                    h('span', Messages.calendar_tempCalendar)
-                ]));
-                makeCalendarEntry(tempCalendars[0], 0);
-                var importTemp = h('button', [
-                    h('i.fa.fa-calendar-plus-o'),
-                    h('span', Messages.calendar_import_temp),
-                    h('span')
-                ]);
-                $(importTemp).click(function () {
-                    importCalendar({
-                        id: tempCalendars[0],
-                        teamId: 0
-                    }, function (err) {
-                        if (err) {
-                            console.error(err);
-                            return void UI.warn(Messages.error);
-                        }
-                    });
-                });
-                if (APP.loggedIn) {
-                    APP.$calendars.append(h('div.cp-calendar-entry.cp-ghost', importTemp));
-                }
-                return;
-            }
```

Comment:
My previous review stated that
> The "temporary calendar" case is not working and was probably not tested

This change makes it so that the feature has been completely removed. I agree that it is indeed not broken anymore, but that's not what we want either !